gallery/src/components/NxAccordion/NxAccordionPage.tsx

```diff
@@ -0,0 +1,167 @@
+
+        Note that this
+        component is stateless – its open state must be tracked externally.
+        See <code className="nx-code">NxStatefulAccordion</code> for a version which tracks its own open state.
+      </p>
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">NxAccordion</h3>
+        </header>
+        <p className="nx-p">
+          In addition to all standard HTML props, <code className="nx-code">NxAccordion</code> can receive
+          the following:
+        </p>
+        <NxTable>
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Name</NxTableCell>
+              <NxTableCell>Type</NxTableCell>
+              <NxTableCell>Required</NxTableCell>
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>onToggle</NxTableCell>
+              <NxTableCell>(boolean => void)</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                A function which gets called when the accordion collapse/expand state is toggled. Receives the new
+                open value (<code className="nx-code">true</code> when
+                opening, <code className="nx-code">false</code> when closing) as an argument.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>open</NxTableCell>
+              <NxTableCell>boolean</NxTableCell>
+              <NxTableCell>Yes</NxTableCell>
+              <NxTableCell>
+                Whether or not the accordion should be rendered "open" with its full content visible, as
+                opposed to collapsed.
+              </NxTableCell>
+            </NxTableRow>
+          </NxTableBody>
+        </NxTable>
+      </section>
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">NxAccordion.Header</h3>
+        </header>
+        <p className="nx-p">
+          <code className="nx-code">NxAccordion.Header</code> can receive standard HTML props.
+        </p>
+      </section>
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">Helper Classes</h3>
+        </header>
+        <p className="nx-p">The following CSS classes are available for use on child elements.</p>
+        <NxTable>
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Name</NxTableCell>
+              <NxTableCell>Location</NxTableCell>
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell><code className="nx-code">nx-accordion__header-title</code></NxTableCell>
```

Comment:
I don't understand why we're bothering with this class when we are effectively setting it to the same values as an `h3`. I also don't understand why we'd go to the trouble of stating that it can be any `h#` value when the class overrides the font-size which is the main visual differentiator between the headings, and as mentioned below the semantics are largely irrelevant.

Reply:
> I don't understand why we're bothering with this class when we are effectively setting it to the same values as an h3.

TBH I hadn't really noticed that it was the same as `nx-h3`.  I'd be alright with getting rid of this custom class and just using `.nx-h3` here.  As far as the note about the sectioning root goes, I was thinking there are multiple places/ways accordions could be used which might fall at different levels in the outline structure.  For instance an accordion which is _not_ wrapped in a tile would likely be at the `h2` level.  That note is essentially to tell the user not to worry about that, because the sectioning root makes it irrelevant.

Reply:
> > I don't understand why we're bothering with this class when we are effectively setting it to the same values as an h3.
>
> TBH I hadn't really noticed that it was the same as nx-h3. I'd be alright with getting rid of this custom class and just using .nx-h3 here. As far as the note about the sectioning root goes, I was thinking there are multiple places/ways accordions could be used which might fall at different levels in the outline structure. For instance an accordion which is not wrapped in a tile would likely be at the h2 level. That note is essentially to tell the user not to worry about that, because the sectioning root makes it irrelevant.

1f32fd51